data/docker-compose-data_download.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+services:
+  marxan-seed-data:
+    build:
+        context: ./data_download
+        dockerfile: Dockerfile
+    volumes:
+    - './data/seed/gadm_3.6/:/gadm_3.6/data/'
+    - './data/seed/wdpa/:/wdpa/data/'
+    - './data/seed/iucn/:/iucn/data/'
+    - './data/seed/world_terrestrial_ecosystems/:/world_terrestrial_ecosystems/data/'
+    user: "$UID:$GID"
+    command: make
+    environment:
```

Comment:
🤔 can we use `env_file` here instead and reference the same `.env` file we use for the API and geoprocessing service? I am not sure id docker compose config files can reference env files in parent directories - if not, I'd say that if a symlink works that could be ok, or even a very rough `cp ../.env .` before running the `docker-compose` command for this component's target in the root `Makefile` - followed by a `rm -f .env` and also adding this "runtime copy" of `.env` to `.gitignore` to avoid confusion.